docs/source/postgresql.md

```diff
@@ -0,0 +1,53 @@
+# PostgreSQL
+
+[PostgreSQL](https://www.postgresql.org/docs/) is a powerful, open source object-relational database system. It is the most [popular](https://survey.stackoverflow.co/2024/technology#most-popular-technologies-database) database by application developers for a few years running. [pgai](https://github.com/timescale/pgai) is a PostgreSQL extension that allows you to easily ingest huggingface datasets into your PostgreSQL database.
+
+
+## Run PostgreSQL with pgai installed
+
+You can easily run a docker container containing PostgreSQL with pgai installed.
+
+```bash
+docker run -d --name pgai -p 5432:5432 \
+-v pg-data:/home/postgres/pgdata/data \
+-e POSTGRES_PASSWORD=password timescale/timescaledb-ha:pg17
+```
+
+Alternatively, you can install pgai into an existing PostgreSQL database. For instructions on how to install pgai into an existing PostgreSQL database, follow the instructions in the [github repo](https://github.com/timescale/pgai).
+
+## Create a table from a dataset
+
+To load a dataset into PostgreSQL, you can use the `ai.load_dataset` function. This function will create a PostgreSQL table, and load the dataset from the Hugging Face Hub
+in a streaming fashion.
+
+```sql
+select ai.load_dataset('squad');
```

Comment:
```suggestion
select ai.load_dataset('rajpurkar/squad');
```

does it also work with namespaced names? (all datasets on HF are namespaced now)